vividus-tests/src/main/resources/story/integration/DragAndDropSteps.story

```diff
@@ -1,10 +1,18 @@
 Meta:
     @epic vividus-plugin-web-app
 
-Scenario: Step verification 'When I drag element located `$origin` and drop it at $location of element located `$target`'
+Scenario: Step verification deprecated 'When I drag element located `$origin` and drop it at $location of element located `$target`'
```

Comment:
please optimize this story to verify both actual and deprecated steps in a single scenario.